<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cebsCalibForm</class>
 <widget class="QWidget" name="cebsCalibForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1194</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CALIBRATION</string>
  </property>
  <widget class="QGroupBox" name="groupBox_calib_set">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>341</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>校准设定</string>
   </property>
   <widget class="QGroupBox" name="groupBox_calib_move_scale">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>运动刻度</string>
    </property>
    <widget class="QRadioButton" name="radioButton_calib_5cm">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>5cm</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_calib_05mm">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0.5mm</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_calib_1cm">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>1cm</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_calib_1mm">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>1mm</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_calib_5mm">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>5mm</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_calib_move_dir">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>运动方向</string>
    </property>
    <widget class="QRadioButton" name="radioButton_calib_y_plus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>上(Y+)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_calib_x_plus">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>右(X+)</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_calib_y_minus">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>下(Y-)</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_calib_x_minus">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>左(X-)</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_calib_move">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>移动</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_calib_left_down">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>设定左下</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_calib_right_up">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>设定右上</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="textEdit_calib_runProgress">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>30</y>
     <width>751</width>
     <height>391</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="autoFormatting">
    <set>QTextEdit::AutoAll</set>
   </property>
   <property name="tabChangesFocus">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&amp;gt;&amp;gt; 2018/5/2 15:30:99 校准开始&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_calib_runProgress">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>校准进展</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_calib_pilot_start">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>校准巡游</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_calib_comp">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>校准完成</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_calib_pilot_stop">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>280</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>巡游停止</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_calib_comp</sender>
   <signal>clicked()</signal>
   <receiver>cebsCalibForm</receiver>
   <slot>slot_calib_close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>383</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>463</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_calib_move</sender>
   <signal>clicked()</signal>
   <receiver>cebsCalibForm</receiver>
   <slot>slot_calib_move()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_calib_left_down</sender>
   <signal>clicked()</signal>
   <receiver>cebsCalibForm</receiver>
   <slot>slot_calib_left_down()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>458</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_calib_right_up</sender>
   <signal>clicked()</signal>
   <receiver>cebsCalibForm</receiver>
   <slot>slot_calib_right_up()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>556</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_calib_pilot_start</sender>
   <signal>clicked()</signal>
   <receiver>cebsCalibForm</receiver>
   <slot>slot_calib_pilot_start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>633</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_calib_pilot_stop</sender>
   <signal>clicked()</signal>
   <receiver>cebsCalibForm</receiver>
   <slot>slot_calib_pilot_stop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>612</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_calib_close()</slot>
  <slot>slot_calib_move()</slot>
  <slot>slot_calib_left_down()</slot>
  <slot>slot_calib_right_up()</slot>
  <slot>slot_calib_pilot_start()</slot>
  <slot>slot_calib_pilot_stop()</slot>
 </slots>
</ui>
